content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -306,41 +274,50 @@ $ curl \
+| `:name`              | Specifies the name of the workspace to update, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization. |
+
+### Request Body
 
+This POST endpoint requires a JSON object with the following properties as a request payload.
 
-### Parameters
+Properties without a default value are required.
 
-Note that workspaces without an associated VCS repository only use the organization name, workspace name, terraform version, and working directory.
+Note that workspaces without an associated VCS repository only use the `auto-apply`, `terraform-version`, and `working-directory`.
 
-- `:organization_name` (`string: <required>`) - Specifies the name of the organization the workspace should belong to. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
-- `name` (`string: <required>`) - Specifies the name of the workspace to update, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.
-- `terraform_version` (`string: <optional>`) - Specifices the version of Terraform to use for this workspace.
-- `working-directory` (`string: ''`) - Specifies the directory that Terraform will execute within. This defaults to the root of your configuration and is typically set to a subdirectory matching the environment when multiple environments exist within the same configuration.
-- `vcs-repo.branch` (`string: ''`) - Specifies the repository branch that Terraform will execute from. If left null or as an empty string, this defaults to the repository's default branch (e.g. `master`) .
-- `vcs-repo.oauth-token-id` (`string: <optional>`) - Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtained from the [oauth-tokens](./oauth-tokens.html) endpoint.
-- `vcs-repo.ingress-submodules` (`boolean: false`) - Specifies whether submodules should be fetched when cloning the VCS repository.
-- `vcs-repo.identifier` (`string: <required>`) - This is the reference to your VCS repository in the format :org/:repo
+| Key path                                      | Type    | Default    | Description                                                                                                                                                                                                                             |
+| --------------------------------------------- | ------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
When I've been reformatting these tables, I've been trimming the final row of hyphens at the end of the word "description," since the extra length doesn't really help anything and looks a little wack. What do you think? I can live with a bit of wackness, especially if we're hoping to automate at least some of this in the future...